Merge and energy consumption pages pre-merge updates (#6599)
ethereum/ethereum-org-website: src/content/upgrades/merge/eth-issuance/index.md

---
title: How The Merge impacts ether supply

> description: Breakdown on how The Merge will impact ether supply

Reviewer: We should consistently try to use ether or ETH on ethereum.org. I feel like ETH makes more sense. 

```suggestion
description: A breakdown of how The Merge will impact ETH supply
```

lang: en
sidebar: true
---

# How The Merge impacts ether supply {#how-the-merge-impacts-ether-supply}

> <Card

Reviewer: - I think we need an introductory paragraph before this card.
- This card could use have a title to orient ("The pre-merge breakdown")

Author: Per discussion, will shift "tldr" box down below some of the intro copy, and add "ETH issuance tldr" as a title to the card. :fire: emoji removed from bullet.

emoji=":chart_decreasing:">

- Mining rewards ~13,000 ETH/day pre-merge
- Staking rewards ~1,600 ETH/day pre-merge
- **After The Merge, only the ~1,600 ETH per day will remain, dropping total new ETH issuance by ~90%**

> - <Emoji text=":fire:" size="1" /> The burn: At an average gas price of at least 16 gwei, at least 1,600 ETH is burned every day, which effectively brings net ETH inflation to zero or less post-merge.

Reviewer: I'm fine with keeping the emoji in the heading below but think it makes this harder to read without adding much. Not strongly opposed to keeping it if you disagree.

</Card>

We can break the supply of ETH into two primary forces: issuance, and burn.

> The **issuance** of ETH is the process of creating ETH that did not previously exist on the balance sheet, while the **burning** of ETH refers to when existing ETH is permanently removed from circulation. The rate of each of these fluctuates based on a number of parameters, and the balance between them determines the resulting inflation/deflation rate of ether.

Reviewer: ```suggestion
The **issuance** of ETH is the process of creating ETH that did not previously exist. The **burning** of ETH is when existing ETH gets destroyed, removing it from circulation. The rate of issuance and burning gets calculated on several parameters, and the balance between them determines the resulting inflation/deflation rate of ether.
```

The way ETH is issued will undergo some changes at time of The Merge. Currently, new ETH is issued from two main sources: the execution layer (EL, i.e. Mainnet) and the consensus layer (CL, i.e. Beacon Chain). After The Merge, issuance from the execution layer will go to zero. Let's break this down.

## Pre-merge {#pre-merge}

> ### EL Issuance {#el-issuance-pre-merge}

Reviewer: I feel pretty strongly that we should spell out execution layer in full. Otherwise, it'll be a headache for translation. Also, two-letter acronyms are relatively uncommon and in this specific case they'll always add cognitive complexity IMO.

Reviewer: Commented elsewhere, but my view is that acronyms are a layer of abstraction. Some research has shown that recently learned acronyms take more mental processing than fully spelt-out terms despite being shorter in length. 

Re: style guide, yeah, it's not ideal. Personally, I'm fine with the more common acronyms (NFTs, dapps, DeFi, DAOs) but when it starts to get less well-known I struggle to justify their use for beginners espescially.

> Under proof-of-work, miners only interact with the EL, and are rewarded in the form of block rewards being the first to solve blocks. Since the [Constantinople upgrade](/history/#constantinople) on Feb-28-2019 this reward has been 2 ETH per block. Miners are also rewarded for publishing [ommer](/glossary/#ommer) blocks, which are valid blocks that don't end up in the longest/canonical chain. These rewards max out at 1.75 ETH per ommer, and are _in addition to_ the reward issued from the canonical block. Mining is an economically intensive activity, which requires high levels of ETH issuance to sustain.

Reviewer: What do you think about removing the dates from here? (but we keep the hyperlinks to the history page) 

Not convinced they add much to explaining issuance.

Author: Compromise by trimming to just the year? Felt like it helped give a bit of orientation to the timeline... roughly how long we've had the current PoW issuance, and roughly how long the PoS issuance has been around.

> Under proof-of-work, miners only interact with the execution layer, and are rewarded in the form of block rewards being the first to solve blocks. Since the [Constantinople upgrade](/history/#constantinople) in 2019 this reward has been 2 ETH per block. Miners are also rewarded for publishing [ommer](/glossary/#ommer) blocks, which are valid blocks that don't end up in the longest/canonical chain. These rewards max out at 1.75 ETH per ommer, and are _in addition to_ the reward issued from the canonical block. Mining is an economically intensive activity, which requires high levels of ETH issuance to sustain.

Reviewer: ```suggestion
On proof-of-work, miners only interact with the execution layer and are rewarded by block rewards if they are the first miner to solve a block. Since the [Constantinople upgrade](/history/#constantinople) in 2019 this reward has been 2 ETH per block. Miners are also rewarded for publishing [ommer](/glossary/#ommer) blocks, which are valid blocks that don't end up in the longest/canonical chain. These rewards max out at 1.75 ETH per ommer and are _in addition to_ the reward issued from the canonical block. Mining is an economically intensive activity requiring high levels of ETH issuance to sustain.
```

Reviewer: Depending on the target audience of for this page, we might want to expand on the ommer definition

### CL Issuance {#cl-issuance-pre-merge}

> The [Beacon Chain](/history/#beacon-chain-genesis) has been live since Dec-01-2020, which is secured by proof-of-stake validators, not miners. This chain was bootstrapped by Ethereum users depositing ETH one-way into a smart contract on Mainnet, which the Beacon Chain listens to, crediting the user with an equal amount on the new chain. Right now these validators are not processing transactions, and are essentially coming to consensus on the state of the validator pool itself.

Reviewer: Same re:dates

> The [Beacon Chain](/history/#beacon-chain-genesis) has been live since 2020, which is secured by proof-of-stake validators, not miners. This chain was bootstrapped by Ethereum users depositing ETH one-way into a smart contract on Mainnet, which the Beacon Chain listens to, crediting the user with an equal amount on the new chain. Right now these validators are not processing transactions, and are essentially coming to consensus on the state of the validator pool itself.

Reviewer: ```suggestion
The [Beacon Chain](/history/#beacon-chain-genesis) went live in 2020. Instead of miners, it is secured by validators using proof-of-stake. This chain was bootstrapped by Ethereum users depositing ETH one-way into a smart contract on Mainnet, which the Beacon Chain listens to, crediting the user with an equal amount on the new chain. Until The Merge happens, the Beacon Chain's validators are not processing transactions and are essentially coming to consensus on the state of the validator pool itself.
```

> Issuance of new ETH on this chain is rewarded to validators for properly attesting to the state of the chain, and is also rewarded when a validator proposes a block. Rewards (or penalties) are calculated and distributed every epoch (6.4 minutes) based on validator performance for that time period. These rewards are _significantly_ lower than the 2 ETH being issued under proof-of-work today every ~13.5 seconds, as operating a validating node is not an economically intense activity, thus does not require or warrant as high a reward.

Reviewer: ```suggestion
Validators on the Beacon Chain are rewarded with ETH for testifying to the state of the chain and proposing blocks. Rewards (or penalties) are calculated and distributed at each epoch (every 6.4 minutes) based on validator performance. The validator rewards are **significantly** less than the miner rewards issued on proof-of-work (2 ETH every ~13.5 seconds), as operating a validating node is not an economically intense activity and thus does not require or warrant as high a reward.
```

### Pre-merge issuance breakdown {#pre-merge-issuance-breakdown}

Total ETH supply: **~119,300,000 ETH** (as of Q2 2022)

> **Execution layer issuance:** <br/>

Reviewer: Why the `<br>`?

Author: Remnant. It's sometimes needed in mdx to get things just right, but this was left over and not needed.

- Estimating at 2.08 ETH per 13.3 seconds\*: **~4,930,000** ETH issued in a year
- Currently inflating at **~4.13%** (4.93M per year / 119.3M total)

> - \*This includes the 2 ETH per canonical block, plus an average of 0.08 ETH over time from ommer blocks. Also uses 13.3 second, the baseline block time target without any influence from a [difficulty bomb](/glossary/#difficulty-bomb). ([See source](https://bitinfocharts.com/ethereum/))

Reviewer: ```suggestion
- \*This includes the 2 ETH per canonical block, plus an average of 0.08 ETH over time from ommer blocks. Also uses 13.3 seconds, the baseline block time target without any influence from a [difficulty bomb](/glossary/#difficulty-bomb). ([See source](https://bitinfocharts.com/ethereum/))
```

**Consensus layer issuance:**

- Using 13,000,000 total ETH staked, the rate of ETH issuance is ~1600 ETH/day ([See source](https://ultrasound.money/))
- Results in **~584,000** ETH issued in a year
- Currently inflating at **~0.49%** (584K per year / 119.3M total)

<InfoBanner>
<strong>Total annual issuance rate: ~4.62%</strong> (4.13% + 0.49%)

**~89.4%** of the issuance is going to miners on the execution layer (4.13 / 4.62 \* 100)

> **~10.6%** is being issued to stakers on the consensus layer (0.49 / 4.62 \* 100)

Reviewer: Not sure why but this is unreadable in darkmode.

<img width="840" alt="Screenshot 2022-06-15 at 15 36 29" src="https://user-images.githubusercontent.com/62268199/173854311-bbc06578-a781-46bd-b0f6-9c5214152938.png">

</InfoBanner>

## Post-merge {#post-merge}

### Execution layer issuance {#el-issuance-post-merge}

> Will be zero. Proof-of-work will no longer be valid under the rules of consensus, and all execution layer activity will be included in "beacon blocks" which are published and attested to by proof-of-stake validators.

Reviewer: If the heading is removed, the content should still make sense.

```suggestion
Execution layer issuance after The Merge will be zero. Proof-of-work will no longer be valid under the rules of consensus. All execution layer activity will be included in "beacon blocks", which are published and attested to by proof-of-stake validators.
```

### Consensus layer issuance {#cl-issuance-post-merge}

> Will continue as it was, with small rewards for attesting-to and proposing blocks. These rewards will continue to accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.

Reviewer 2: Should we add a reference to the Shanghai upgrade on https://ethereum.org/en/history/ ?

Author: We haven't even put The Merge up there yet (aka Paris/Bellatrix forks), but it would at least give us a place to link to internally, so I'm all for it. This should be in a separate PR though, can add a card for this on Notion.

> Will continue as it was, with small rewards for attesting-to and proposing blocks. These rewards will continue to accrue to _validator balances_ that are managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.

Reviewer: ```suggestion
Consensus layer issuance will continue as before The Merge, with small rewards for validators who attest to and propose blocks. Validator rewards will continue to accrue to _validator balances_ that get managed within the consensus layer. These are separate Ethereum accounts to the accounts we're used to on Mainnet, and until the Shanghai upgrade funds from validator accounts will not be withdrawable/transferrable. This means that although new ETH is still being issued, 100% of it will be locked from the market until this upgrade occurs. When the Shanghai upgrade is rolled out, this ETH will become available.
```

> When validator withdrawals are enabled, stakers will be incentivized to remove their _earnings/rewards (balance over 32)_ as these funds are otherwise not contributing to their stake weight (which maxes as 32).

Reviewer: ```suggestion
When validator withdrawals are enabled, validators will be incentivized to remove their _earnings/rewards (balance over 32 ETH)_ as these funds are otherwise not contributing to their stake weight (which maxes as 32 ETH).
```

Author: Note on this, using term "stakers" intentionally when referring to the humans behind the validator accounts.

Also, agree with the first update here ("balance over 32 ETH") but the second is blurrier, since the weight of a validator vote (effective balance) is not strictly "denominated" in ETH... It's a whole number integer value that is calculated based on the ETH balance, but there are some nuances. 

I'm going to update the first, but leave the second without. Feel free to push back, but that is my reasoning.

> Stakers may also choose to exit and withdraw their entire validator balance, but it is important to remember that full validator exits are rate limited. Depending on the total ETH staked at the time, only 6 validators may exit in a given epoch (6.4 minute period). This decreases to as low as 4 as more validators leave, to intentionally prevent the ability for large amounts of staked ETH to leave at once.

Reviewer: ```suggestion
Stakers may also choose to exit and withdraw their entire validator balance. To ensure Ethereum is stable, the number of validators leaving simultaneously is capped. Only six validators may exit in a given epoch (6.4 minute period) depending on the total ETH staked at the time. This decreases to as low as four as more validators withdraw to intentionally prevent large destabilizing amounts of staked ETH from leaving at once.
```

Author: Missed this before, will add

### Post-merge inflation breakdown {#post-merge-inflation-breakdown}

- Total ETH supply: **~119,300,000 ETH** (as of Q2 2022)
- Execution layer issuance: **0**
- Consensus layer issuance: Same as above, **~0.49%** annual issuance rate (with 13 million ETH staked)
- Total annual issuance rate: **~0.49%**

<InfoBanner>
Total annual issuance rate: <strong>~0.49%</strong>
Net reduction in annual ETH issuance: <strong>~89.4%</strong> (0.49% / 4.62% * 100)
</InfoBanner>

## <Emoji text=":fire:" size="1" /> The burn {#the-burn}

> This flip side of ETH issuance is the rate at which ETH is burned. To execute a transaction on Ethereum, a minimum fee (known as a `base fee`) must be paid, which fluctuates depending on network activity in the moment. This fee is paid in ETH, is _required_ for the transaction to be considered valid, and is _burned_ in the process, removing it from circulation.

Reviewer: Wasn't clear on the meaning of 'this flip side', I think this is what was meant?

```suggestion
The other thing to consider about ETH issuance is the rate at which ETH is burned. For a transaction to execute on Ethereum, a minimum fee (known as a `base fee`) must be paid, which fluctuates depending on network activity. The fee is paid in ETH and is _required_ for the transaction to be considered valid. The fee gets _burned_ during the transaction process, removing it from circulation.
```

Author: Kept most updates here, adjusted to "The opposite force to ETH issuance" since I already introduced the idea of opposing forces at the beginning.

<InfoBanner>

> Fee burning went live with the <a href="/history/#london">London upgrade</a> in August 2021, and will not change after the Merge.

Reviewer: ```suggestion
Fee burning went live with <a href="/history/#london">the London upgrade</a> in August 2021, and will continue after the Merge.
```

</InfoBanner>

> On top of the fee burn implemented by the London upgrade, validators can also incur penalties for being offline, or worse they can be slashed for breaking specific rules that threaten network security. These penalties result in a reduction of ETH from that validator's balance, which is not directly rewarded to any other account, effectively burning it from circulation.

Reviewer: ```suggestion
On top of the fee burn implemented by the London upgrade, validators can also incur penalties for being offline, or worse, they can be slashed for breaking specific rules that threaten network security. These penalties result in a reduction of ETH from that validator's balance, which is not directly rewarded to any other account, effectively burning it from circulation.
```

## Further reading {#further-reading}

> - [Ultrasound.money](https://ultrasound.money/) - Dashboards available to visualize ETH issuance and burn in real time

Reviewer: ```suggestion
- [Ultrasound.money](https://ultrasound.money/) - Dashboards available to visualize ETH issuance and burn in real-time
```
